src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerIntellisenseFilter.cs

```diff
@@ -146,20 +128,15 @@ void executeNextCommandTarget()
                             // target isn't the one we want, because we've
                             // definitely remapped buffers. Ask our context for
                             // the real subject buffer.
-                            ExecuteInvokeCompletionList(_context.Buffer, _context.ContentType, () =>
-                            {
-                                // We cannot just pass executeNextCommandTarget becuase it would execute TYPECHAR
-                                var showMemberListCmdGroupId = VSConstants.VSStd2K;
-                                NextCommandTarget.Exec(ref showMemberListCmdGroupId, (uint)VSConstants.VSStd2KCmdID.SHOWMEMBERLIST,
-                                    executeInformation, pvaIn, pvaOut);
-                            });
+                            NextCommandTarget.Exec(VSConstants.VSStd2K, (uint)VSConstants.VSStd2KCmdID.SHOWMEMBERLIST,
+                                executeInformation, pvaIn, pvaOut);
                         }
                     }
 
                     break;
 
                 default:
-                    return base.ExecuteVisualStudio2000(ref pguidCmdGroup, commandId, executeInformation, pvaIn, pvaOut, subjectBuffer, contentType);
+                    return base.ExecuteVisualStudio2000(ref pguidCmdGroup, commandId, executeInformation, pvaIn, pvaOut);
```

Comment:
Yes, this appears to be a bug where this would return and we'd never do the SetStateFlags below. Leaving the bug here for now.